.eslintrc

```diff
@@ -5,10 +5,12 @@
     "es6": true,
     "node": true
   },
+  "parser": "babel-eslint",
```

Comment:
what do we need `babel-eslint` for?

Reply:
i think I needed for testing. Can confirm later, just need to remove and check

Reply:
@n3op2 did we check this. We don't need it anywhere else so I'd be very surprised

Reply:
removed, as it was working, I do remember that there was reason for this, but can not remember what, probably when I was using some other libs e.g. for stubbing or so.